### DISCUSSION TITLE
Should I move the ungoogled-chromium folder on Windows?

### DISCUSSION THREAD

**Discussion by FriedGenera** [Eloston/ungoogled-chromium#2612] [Q&A]
I'm trying to build ungoogeled-chromium on windows but I keep running into this error -
`GetFullPathNameA(gen/third_party/blink/renderer/bindings/modules/v8/v8_union_gpucanvascontext_imagebitmaprenderingcontext_offscreencanvasrenderingcontext2d_webgl2renderingcontext_webglrenderingcontext.h): The filename or extension is too long.
ninja: build stopped: subcommand failed.`

I have already set the max_path registry key to 1, should I move the file from my desktop or is there another solution?

**Comment by therentabrain** [comment]
Resurrecting an old thread but it's an easy answer:
Yes.
The depth of subdirectories required is in addition to (by default)  c:\users\yourusername\desktop\ so by moving it closer to the root of the drive, you'll be reducing that depth/length.
Also, you could use subst to create a temporary drive letter which can reduce the length.

(I hope you know that you don't need to build your own ungoogled chromium if you download and run premade trusted binaries.)
